lib/resty/core/base.lua

```diff
@@ -174,6 +209,20 @@ function _M.ref_in_table(tb, key)
 end
 
 
+function _M.check_subsystem(...)
+    local subsystems = {...}
```

Comment:
Can we get rid of this Lua table? We can just use `select(i, ...)` to index individual arguments.